pkg/cmd/roachtest/roachtestutil/mixedversion/planner_test.go

```diff
@@ -339,6 +339,17 @@ func newRand() *rand.Rand {
 
 func newTest(options ...CustomOption) *Test {
 	testOptions := defaultTestOptions()
+	defaultTestOverrides := []CustomOption{
+		// Enforce system-only deployments by default in tests; those that
+		// test multitenant deployments specifically should pass the
+		// `EnabledDeploymentModes` option explicitly.
+		EnabledDeploymentModes(SystemOnlyDeployment),
```

Comment:
Should we add `DisableSkipVersionUpgrades` to this and get rid of [the override here](https://github.com/cockroachdb/cockroach/blob/0f2ffbf8ccd3376dc04274a3d738a79626e4a26c/pkg/cmd/roachtest/roachtestutil/mixedversion/planner_test.go#L89)?

Reply:
Good point, done.